tests/legacy-cli/e2e/utils/assets.ts

```diff
@@ -4,7 +4,8 @@ import {getGlobalVariable} from './env';
 import {relative} from 'path';
 import {copyFile, writeFile} from './fs';
 import {useBuiltPackages} from './project';
-import { git, silentNpm } from './process';
+import { git, silentNpm, silentYarn } from './process';
+import { skip } from 'rxjs/operators';
```

Comment:
`skip` doesn't appear to be used.

Reply:
Got it. Will force push a fix for this once a decision is made on the other comment.